scripts/tasks/lint-imports.js

```diff
@@ -97,20 +123,37 @@ module.exports = function(options) {
   }
 
   function _evaluateImport(filePath, importPath, importErrors) {
+    let fullImportPath;
     if (importPath.indexOf('.') === 0) {
       // import is a file path. is this a file?
-      const fullImportPath = _evaluateImportPath(path.dirname(filePath), importPath);
+      fullImportPath = _evaluateImportPath(path.dirname(filePath), importPath);
+    } else {
+      const pkgNameMatch = importPath.match(pkgNameRegex);
+      if (pkgNameMatch === null) {
+        return;
```

Comment:
What does this mean? There was no matching package found?

Reply:
Added a comment in the code that explains what it means.